scipy-1.0/subpackages.tex

```diff
@@ -107,13 +107,16 @@
     combinatorial functions \texttt{comb} and \texttt{factorial};
     and more.
 \item[\texttt{stats}]
-    The \texttt{stats} subpackage provides a large collection of continuous
-    and discrete probability distributions, each with methods to compute
+    The \texttt{stats} subpackage provides: a large collection of continuous
```

Comment:
Colon is not appropriate here according to Rule 2 (https://www.grammarbook.com/punctuation/colons.asp)